RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/extractors/LegacyFalseRefactoringsExtractor.java

```diff
@@ -147,9 +129,9 @@ void traverse(Repository repo, String filePath, CompilationUnit root, Node cur)
     }
 
     void traverseBlock(Repository repo, String filePath, CompilationUnit root, MethodDeclaration md, Statement blk, int dep, int k) throws GitAPIException {
-        if (!canExtract(blk, dep)) {
+       /* if (!canExtract(blk, dep)) {
```

Comment:
why it is commented now?

Reply:
was testing this feature in legacy extractor, reverted this change